src/stores/WidgetsStore.ts

```diff
@@ -806,6 +822,63 @@ export class WidgetsStore {
         }
     };
 
+    @action onImagePanelButtonClick = () => {
```

Comment:
perhaps we could remove `@action` here because we don't directly modify the observables

Reply:
Yes, I agree. We only need `@action` decorators when we do either the following:
- Modify `@observable` properties directly
- Call multiple `@action` functions from within the same function. Decorating the calling function with `@action` means that any `@computed` or `autorun` dependencies will only be run once, _after_ the calling function has exited.

Reply:
thanks for your good reminder